app/src/main/kotlin/com/fernandocejas/sample/features/movies/MoviesFragment.kt

```diff
@@ -55,7 +56,9 @@ class MoviesFragment : BaseFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         initializeView()
-        loadMoviesList()
+        if (savedInstanceState == null) {
```

Comment:
Verify that the data loads properly after process death https://stackoverflow.com/a/49107399/2413303

Reply:
you are right, any idea how to solve this?